### DISCUSSION TITLE
Is the size limit of V5 the same as size limit of V4?

### DISCUSSION THREAD

**Discussion by ioagr** [litedb-org/LiteDB#2231] [Q&A]
Hello! Is the size limit of V5 the same as size limit of V4 (16TB)?
Or it is UInt.MaxValue * PageSize (8192) = 32TB?

**Comment by LMeinhardt** [comment]
According the provided documention:

Database Max Length: In theory, UInt.MaxValue * PageSize (4096) = 16TB
